notebooks/algorithms/community/Triangle-Counting.ipynb

```diff
@@ -254,9 +211,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# create a Graph using the source (src) and destination (dst) vertex pairs from the Dataframe \n",
-    "G = cugraph.Graph()\n",
-    "G.from_cudf_edgelist(gdf, source=\"0\", destination=\"1\",edge_attr=\"weight\")"
+    "G = karate.get_graph()\n",
+    "G = G.to_undirected()"
```

Comment:
This change is perhaps out-of-scope for this PR, but it would be nice if `get_graph` accepted a `create_using` option similar to `Graph.from_edgelist` that allowed you to get a directed or undirected graph.